rspecq.gemspec

```diff
@@ -19,6 +19,7 @@ Gem::Specification.new do |s|
   end
 
   s.add_dependency "redis"
+  s.add_dependency "sentry-raven"
```

Comment:
Just a note that although raven-ruby has [only one dependency at the moment](https://github.com/getsentry/raven-ruby/blob/d9d41784c3b5888295bf4b013416f132045b22d4/sentry-raven.gemspec#L20) as far as I can tell, this could create unnecessary issues with gem incompatibilities when the integration is not used. So perhaps in the future the whole integration along with dependencies can become opt-in?

Reply:
Maybe we could do this by not adding sentry-raven as a dependency and requiring the user to explicitly add it to their Gemfile. That said, I believe we should hold on until this pops up as an issue.